pkg/constants/constants.go

```diff
@@ -91,6 +91,9 @@ const (
 	// Bundle ref for a buildah-remote build
 	CUSTOM_BUILDAH_REMOTE_PIPELINE_BUILD_BUNDLE_ENV string = "CUSTOM_BUILDAH_REMOTE_PIPELINE_BUILD_BUNDLE"
 
+	//Bundle ref for custom source-build
+	CUSTOM_SOURCE_BUILD_PIPELINE_BUNDLE_ENV string = "CUSTOM_SOURCE_BUILD_PIPELINE_BUNDLE"
```

Comment:
What value should be set to this env var? Could you please commit it out?

Reply:
Added now.